src/audio_worklet.js

```diff
@@ -164,23 +145,12 @@ class BootstrapMessages extends AudioWorkletProcessor {
     // scope to create the real AudioWorkletProcessors that call out to Wasm to
     // do audio processing.
     let p = globalThis['messagePort'] = this.port;
```

Comment:
Now that it's all in the same file, we should be able to replace this `globalThis[...]` property here and its usages in libwebaudio.js with a simple variable, which should also compress better.